apps/web/playwright/fixtures/users.ts

```diff
@@ -311,7 +313,7 @@ const createUser = (workerInfo: WorkerInfo, opts?: CustomUserOpts | null): Prism
   const uname = `${opts?.username || "user"}-${workerInfo.workerIndex}-${Date.now()}`;
   return {
     username: uname,
-    name: opts?.name,
+    name: opts?.name || "Default Name",
```

Comment:
maybe we should add a way to not add a name, is that ever required for tests or even possible for real users?

Reply:
At least one test relies on a "Nameless" return. Will change the default to that for now.